index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Page</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <header>
+        <div class="status">
+            <span id="time">9:41</span>
+            <span id="icons">
+                <img src="assets/images/Cellular.png" alt="" class="icon">
+                <img src="assets/images/Wifi.png" alt="" class="icon">
+                <img src="assets/images/Battery.png" alt="" class="icon">
+            </span>
+        </div>
```

Comment:
- [ ] The mobile phone navigation is not a part of the webpage. Kindly remove it. 😄

![image](https://user-images.githubusercontent.com/92215878/204811389-b20cbcb6-5da3-496f-9164-0f344762c9c3.png)